addons/longhorn/1.1.2/install.sh

```diff
@@ -190,6 +138,48 @@ function validate_longhorn_ds() {
     fi
 }
 
+# if this is a multinode cluster, we need to prepare all hosts to run the daemonset
+function maybe_init_hosts() {
```

Comment:
I would assume that this function needs to be added to the template too

Reply:
added